java/java-cloud-run-hello-world/cloudbuild.yaml

```diff
@@ -0,0 +1,64 @@
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}', '.']
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}']
+
+- id: 'Deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  args:
+  - 'run'
+  - 'deploy'
+  - '${_SERVICE}'
+  - '--image'
+  - 'gcr.io/${PROJECT_ID}/${_SERVICE_TAG}:${COMMIT_SHA}'
+  - '--allow-unauthenticated'
+  - '--region'
+  - '${_CLOUDSDK_REGION}'
+  - '--platform'
+  - 'managed'
+
+- id: 'Get Cloud Run URL'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    get_url() {
+        gcloud run services describe ${_SERVICE} --format 'value(status.url)' \
+          --platform managed --region ${_CLOUDSDK_REGION}
+    }
+    echo $(get_url) > _service_url
+    echo "Cloud Run URL for ${_SERVICE} is $(cat _service_url)"
+- id: 'Integration Tests'
+  name: 'gcr.io/cloud-builders/curl:latest'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    set -ex
+    keyword='Congratulations, you successfully deployed a container image to Cloud Run'
+    chmod +x test_content.sh
+    ./test_content.sh -r 25 -i 3 -u $(cat _service_url) -k $keyword
```

Comment:
Where is test_content.sh file? I assume that the intention was to wrap invocation of the tests in this file but didn't find it as part of this change.